packages/pwa/app/pages/product-list/partials/empty-results.jsx

```diff
@@ -9,10 +9,15 @@ import React, {Fragment} from 'react'
 import {Button, Text, Flex, Stack, Link} from '@chakra-ui/react'
 import PropTypes from 'prop-types'
 import {Link as RouteLink} from 'react-router-dom'
-import {FormattedMessage, useIntl} from 'react-intl'
+import {defineMessage, FormattedMessage, useIntl} from 'react-intl'
 import {SearchIcon} from '../../../components/icons'
 import RecommendedProducts from '../../../components/recommended-products'
 
+const contactUsMessage = defineMessage({
```

Comment:
is there a reason why we treat this message a bit different by using `defineMessager`?

Reply:
I wanted to refactor duplicate messages. So here we are defining it only once.

Reply:
ah I see, that makes sense 👍